### ISSUE TITLE
The result when passing an array of integer to srem is not as expected

### ISSUE THREAD

**Issue by glasses618** [sds/mock_redis#165]
Hi,
When I pass an array of integer to `srem`, the members could not be deleted as expeced. 
But in `redis` gem, it works.

```ruby
r = Redis.new;
r.sadd('k', [1]);
r.srem('k', [1])
=> 1

m = MockRedis.new;
m.sadd('k', [1]);
m.srem('k', [1])
=> 0
```
#### Environment
ruby 2.5.5
redis 4.1.0
mock_redis 0.20.0

**Comment by sds** [comment]
Thanks for the report! 

Would happily accept a pull request with a test which fixes this behavior. Thanks!

**Comment by sds** [comment]
Fixed in #166.
